### ISSUE TITLE
Page.get_pixmap(clip=Rect()) bug in rebased fitz_new version

### ISSUE THREAD

**Issue by ellacroix** [pymupdf/PyMuPDF#2692]
## Describe the bug (mandatory)
The function Page.get_pixmap(clip=fitz.Rect()), when adding a fitz.Rect() as argument, doesn't work in the new rebased fitz_new version.
It raises this exception:
``` NameError: name 'fz_identity' is not defined ```

## To Reproduce (mandatory)
To reproduce, you need a Python environment with Tesseract installed.

``` 
import os
# os.environ['TESSDATA_PREFIX'] = 'C:\\work\\apps\\Tesseract-OCR\\tessdata'
import fitz_new as fitz
# import fitz

doc = fitz.Document('notice_8676.pdf')
for page in doc:
	pix = page.get_pixmap(clip=fitz.Rect(0,0,10,10))
	print(pix)
```

[notice_8676.pdf](https://github.com/pymupdf/PyMuPDF/files/12729206/notice_8676.pdf)

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[14], line 8
      6 doc = fitz.Document('notice_8676.pdf')
      7 for page in doc:
----> 8 	pix = page.get_pixmap(clip=fitz.Rect(0,0,10,10))
      9 	print(pix)

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\utils.py:899](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/utils.py:899), in get_pixmap(page, matrix, dpi, colorspace, clip, alpha, annots)
    896     raise ValueError("unsupported colorspace")
    898 dl = page.get_displaylist(annots=annots)
--> 899 pix = dl.get_pixmap(matrix=matrix, colorspace=colorspace, alpha=alpha, clip=clip)
    900 dl = None
    901 if dpi:

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\__init__.py:2450](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/__init__.py:2450), in DisplayList.get_pixmap(self, matrix, colorspace, alpha, clip)
   2448 if not colorspace:
   2449     colorspace = mupdf.FzColorspace(mupdf.FzColorspace.Fixed_RGB)
-> 2450 val = JM_pixmap_from_display_list(self.this, matrix, colorspace, alpha, clip, None);
   2451 val.thisown = True
   2452 return val

File [c:\work\apps\python_env\main\Lib\site-packages\fitz_new\__init__.py:16228](file:///C:/work/apps/python_env/main/Lib/site-packages/fitz_new/__init__.py:16228), in JM_pixmap_from_display_list(list_, ctm, cs, alpha, clip, seps)
  16226 if not mupdf.fz_is_infinite_rect(rclip):
  16227     dev = mupdf.fz_new_draw_device_with_bbox(matrix, pix, irect)
> 16228     mupdf.fz_run_display_list(list_, dev, fz_identity, rclip, None)
  16229 else:
  16230     dev = mupdf.fz_new_draw_device(matrix, pix)

NameError: name 'fz_identity' is not defined
```

## Expected behavior (optional)
Page.get_pixmap(clip=Rect()) should return a pixmap corresponding to the given Rect.

## Your configuration (mandatory)
 - Operating system, potentially version and bitness: **win32**
 - Python version, bitness: **3.11.3 (tags/v3.11.3:f3909b8, Apr  4 2023, 23:49:59) [MSC v.1934 64 bit (AMD64)]**
 - PyMuPDF version, installation method (**wheel** or **generated** from source): **1.23.3, installed via pip**

**Comment by julian-smith-artifex-com** [comment]
Thanks for this report, i have a fix in my tree.

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.23.4.
